config/install_debug.yaml

```diff
@@ -7434,16 +7436,6 @@ rules:
   - update
   - patch
   - delete
-- apiGroups:
-  - quota
-  resources:
-  - resourcequotas
-  - limitranges
-  verbs:
-  - create
-  - update
-  - patch
-  - delete
```

Comment:
Would it be more clear to separate these two resources from `quota` apiGroup?

Reply:
There’s no quota api group ;)

Reply:
Remember https://github.com/kyverno/kyverno/pull/3336/files

Reply:
It's just that the file hadn't been regenerated.